importers/test/test_djornl_parser.py

```diff
@@ -60,20 +71,11 @@ def test_load_empty_files(self):
     def test_load_missing_files(self):
         """ test loading when files cannot be found """
 
-        # this dir does not contain the correct file structure
-        # path: test/djornl/empty_files/cluster_data
-        RES_ROOT_DATA_PATH = os.path.join(_TEST_DIR, 'djornl', 'empty_files', 'cluster_data')
-        parser = self.init_parser_with_path(RES_ROOT_DATA_PATH)
-
-        err_str = "No such file or directory: '" + RES_ROOT_DATA_PATH
-        with self.assertRaisesRegex(FileNotFoundError, err_str):
-            parser.load_edges()
-
-        with self.assertRaisesRegex(FileNotFoundError, err_str):
-            parser.load_node_metadata()
-
-        with self.assertRaisesRegex(FileNotFoundError, err_str):
-            parser.load_cluster_data()
+        RES_ROOT_DATA_PATH = os.path.join(_TEST_DIR, 'djornl', 'missing_files')
+        # not found
+        err_str = os.path.join(RES_ROOT_DATA_PATH, "edges.tsv") + ': file does not exist'
+        with self.assertRaisesRegex(RuntimeError, err_str):
+            self.init_parser_with_path(RES_ROOT_DATA_PATH)
```

Comment:
all the validation now happens before file parsing starts, so the individual tests in the previous version can be replaced by this single test. Woohoo!